release_planning/puppet_module_augmenter.rb

```diff
@@ -0,0 +1,20 @@
+class PuppetModuleAugmenter
```

Comment:
this is not used by anything outside of the release scripts. it should just live in the release planning script